frontend/package.json

```diff
@@ -15,6 +15,7 @@
     "js-cookie": "^2.2.0",
     "moment": "^2.24.0",
     "react": "^16.4.1",
+    "react-app-polyfill": "^1.0.4",
```

Comment:
You will also need to add this in the other package.json file we have for `heroku`. `/phishtray/package.json`. Just copy and paste the module into the file and that's it